integr8ly/integreatly-operator
# MGDAPI-1261: Automate H22 test (rate limit redis counter)

# Description

> Link to JIRA: https://issues.redhat.com/browse/MGDAPI-1261

Automate test H22 using Golang, create script to launch testing and modify test steps to indicate script usage and expected output.

Extend `ThreeScaleInterface` with more API calls that are used by the test to create an API

> _Now with 500% more emojis in test output 🔥_

## Verification steps

1. Install RHOAM on a cluster
2. Perform the updated steps for H22 and ensure the test finishes successfully:
    * https://github.com/sergioifg94/integreatly-operator/blob/MGDAPI-1261/test-cases/tests/products/h22-validate-that-rate-limit-service-is-working-as-expected.md

## Type of change




## Checklist
